variants/ancientmediterranean/ancientmediterranean.go

```diff
@@ -27,7 +27,7 @@ var AncientMediterraneanVariant = common.Variant{
 	Start:      AncientMediterraneanStart,
 	Blank:      AncientMediterraneanBlank,
 	Phase:      classical.Phase,
-	Parser:     orders.ClassicalParser,
+	Parser:     orders.Parser,
```

Comment:
Actually - I should change this back again. It doesn't sound at all classical now.